adoc/admin-ses-integration.adoc

```diff
@@ -0,0 +1,594 @@
+type: "kubernetes.io/rbd"
+data:
+  key: "$(echo CEPH_SECRET | base64)"
+*EOF*`` 
+----
+. Create a volume on the SES cluster: 
++
+
+----
+{prompt.root}``rbd create -s SIZE YOUR_VOLUME`` 
+----
++
+Replace `SIZE` with the size of the image, for example `2G` (2 Gigabyte), and `YOUR_VOLUME` is the name of the image. 
+. Create the persistent volume on the {master_node} : 
++
+
+----
+{prompt.user}``kubectl apply -f - << *EOF*
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: PV_NAME
+spec:
+  capacity:
+    storage: SIZE
+  accessModes:
+    - ReadWriteOnce
+  rbd:
+    monitors:
+    - 'MONITOR1_IP:MONITOR1_PORT'
+    - 'MONITOR2_IP:MONITOR2_PORT'
+    - 'MONITOR3_IP:MONITOR3_PORT'
+    pool: RDB_POOL
+    image: YOUR_VOLUME
+    user: admin
+    secretRef:
+      name: ceph-secret
+    fsType: ext4
+    readOnly: false
+*EOF*`` 
+----
++
+Replace `SIZE` with the desired size of the volume.
+Use the _gibibit_ notation, for example ``2Gi``. 
+. Create a persistent volume claim on the {master_node} : 
++
+
+----
+{prompt.user}``kubectl apply -f - << *EOF*
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: PVC_NAME
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: SIZE
+*EOF*`` 
+----
++
+Replace `SIZE` with the desired size of the volume.
+Use the _gibibit_ notation, for example ``2Gi``. 
++
+.Listing Volumes
+NOTE: This persistent volume claim does not explicitly list the volume.
+Persistent volume claims work by picking any volume that meets the criteria from a pool.
+In this case we specified any volume with a size of 2G or larger.
+When the claim is removed the recycling policy will be followed. 
++
+
+. Create a pod that uses the persistent volume claim. On the {master_node} run the following: 
++
+
+----
+{prompt.user}``kubectl apply -f - <<*EOF*
+apiVersion: v1
+kind: Pod
+metadata:
+  name: POD_NAME
+spec:
+  containers:
+  - name: CONTAINER_NAME
+    image: IMAGE_NAME
+    volumeMounts:
+    - mountPath: /mnt/rbdvol
+      name: rbdvol
+  volumes:
+  - name: rbdvol
+    persistentVolumeClaim:
+      claimName: PV_NAME
+*EOF*`` 
+----
+. Verify that the pod exists and its status. On the {master_node} run: 
++
+
+----
+{prompt.user}``kubectl get po`` 
+----
+. Once pod is running, check the volume by running on the {master_node} : 
++
+
+----
+{prompt.user}``kubectl exec -it CONTAINER_NAME -- df -k`` ...
+/dev/rbd3               999320      1284    929224   0% /mnt/rbdvol
+...
+----
+
+
+In case you need to delete the pod, run the following command on the {master_node}: 
+
+----
+{prompt.user}``kubectl delete pod CONTAINER_NAME`` 
+----
+
+.Deleting A Pod
+[NOTE]
+====
+When you delete the pod, the persistent volume claim is deleted as well.
+The RBD is not deleted. 
+====
+
+== Using RBD for Dynamic Persistent Volumes
+
+
+The following procedure describes how to attach a pod to a RDB dynamic persistent volume. 
+
+.Procedure: Creating a Pod with RBD in Persistent Volume
```

Comment:
It should be Procedure: Creating a Pod with RBD in `Dynamic` Persistent Volume

Reply:
Fixed, thanks.